test/html.js

```diff
@@ -300,8 +300,8 @@ describe('HTML Parser', function () {
 				var root = parseHTML('<p a=12 data-id="!$$&amp;" yAz=\'1\'></p>');
 				root.firstChild.rawAttributes.should.eql({
 					'a': '12',
-					'data-id': '"!$$&amp;"',
```

Comment:
I think both `attributes` and `rawAttributes` exist just because the `encode/decode` thing, and I don't know why should we encde/decode the attribute value.
In my opinion, rawAttributes is the origin value we put in, with or without quotes. so I don't think we should change this back, do you agree?

Reply:
We should `decode` the attribute value, because browsers work like that and I saw bunch of html with encoded attributes. Eg html can look like this `<img src="default.jpg" alt="Verissimo, Ilaria D&#39;Amico: &laquo;Sogno una bambina. Buffon mi ha chiesto in moglie tante volte&raquo;">`.

`rawAttributes ` doesn't exist in browsers, so there is no clear definition, but  quotes are there to guide the html parsers and by no means it's part of the attribute value.